superset-frontend/package.json

```diff
@@ -113,7 +113,7 @@
     "antd": "^4.9.4",
     "array-move": "^2.2.1",
     "babel-plugin-typescript-to-proptypes": "^2.0.0",
-    "bootstrap": "^3.4.1",
+    "bootstrap": "^5.1.1",
```

Comment:
bootstrap is only used to enable menus from FAB to work. So 2 questions:
1. Do we still need this, or have we replaced the FAB menus everywhere?
2. Do we need to ensure FAB is using bootstrap v5 before we make this upgrade?

Reply:
cc @dpgaspar

Reply:
The examples are using 4.2.1, perhaps we need that here too: https://github.com/dpgaspar/Flask-AppBuilder/blob/27b15e59316e85e0fe62b8aa9978391ed4c729c9/examples/react-rest-api/app/static/package.json#L7

Not sure if that has the vuln fixed in it or not

Reply:
> The examples are using 4.2.1, perhaps we need that here too: https://github.com/dpgaspar/Flask-AppBuilder/blob/27b15e59316e85e0fe62b8aa9978391ed4c729c9/examples/react-rest-api/app/static/package.json#L7
> 
> Not sure if that has the vuln fixed in it or not
According to  CVE-2019-8331 the issue fixed in version 4.3.1

Reply:
We removed bootstrap from the frontend, are we exposing this package anywhere or just using the styles? I only see references to it in `package.*.json`

Reply:
@etr2460 If we still use it, can we bump up the version to a secure version? For both jquery and bootstrap?